beacon_node/network/src/beacon_processor/work_reprocessing_queue.rs

```diff
@@ -770,6 +856,29 @@ impl<T: BeaconChainTypes> ReprocessQueue<T> {
                     }
                 }
             }
+            InboundEvent::ReadyBackfillSync(queued_backfill_batch) => {
+                let duration_from_slot_start = slot_clock.millis_from_current_slot_start();
+
+                debug!(
+                    log,
+                    "Sending scheduled backfill work";
+                    "duration_from_slot_start" => format!("{:?}", duration_from_slot_start)
```

Comment:
```suggestion
                    "millis_from_slot_start" => format!("{:?}", duration_from_slot_start)
```

Just so we know the unit. I might have even made this suggestion, apologies if so.

Reply:
Updated this to convert duration to milliseconds. Thanks!